tinytodo-go/test_tinytodo.py

```diff
@@ -0,0 +1,76 @@
+from tinytodo import *
```

Comment:
I assume that the go version uses the same "frontend" Python file as the Rust version.. If so, we should also use one copy only.

Reply:
Agreed, although it's not totally clear to me what value the Python frontend adds over just running `go build` and `cmd/server` directly. Maybe it lets you interact with the server from a REPL somewhat easily? Dunno.

Reply:
Yes. We use the Python REPL to interact with the backend server.